### ISSUE TITLE
WebGUI locks up or fails to render properly 

### ISSUE THREAD

**Issue by hissingshark** [hyperion-project/hyperion.ng#777]
- [x] I confirm that this is an issue rather than a question.

## Bug report
WebGUI locks up or fails to render properly when selecting `Configuration -> Capturing Hardware`.

#### Steps to reproduce
Just connect to the GUI as normal.  Tested on Chrome via 2 x desktops and 1 x mobile.
Select `Configuration -> Capturing Hardware`.

#### What is expected?
Normal working appearance.

#### What is actually happening?
All other options render normally.

But if `Configuration -> Capturing Hardware` is selected it appears blank apart from the title.  Attempting to refresh the page brings up the Hyperion logo overlay and it locks up.
![Screenshot_20200424-073459](https://user-images.githubusercontent.com/20879507/80192023-76432f00-860e-11ea-8365-5752aba7a462.png)
Browser tab must be closed and GUI reconnected to try again.

If you don't attempt to refresh and instead select other options they are broken too
![Screenshot_20200424-073615](https://user-images.githubusercontent.com/20879507/80192072-8ce98600-860e-11ea-9d31-287b8e38280d.png)
 - although they can be fixed by refreshing the page.

#### System
Hyperion Server: 
- Build:       master (GitHub-072bee1d/3a2b58b5-1587481685)
- Build time:  Apr 23 2020 09:29:20
- Git Remote:  https://github.com/hyperion-project/hyperion.ng.git
- Version:     2.0.0-alpha.5
- UI Lang:     auto (BrowserLang: en-GB)
- UI Access:   expert
- Avail Capt:  framebuffer,amlogic

Hyperion Server OS: 
- Distribution: Open Source Media Center
- Arch:         arm64
- Kernel:       linux (4.9.113-16-osmc (WS: 32))
- Browser:      Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/80.0.3987.106 Safari/537.36

**Comment by brindosch** [comment]
Hello,
can you reproduce this issue on a desktop/laptop browser and open the developer console of the browser to show the error message?

Thank you

**Comment by hissingshark** [comment]
With `Configuration -> Capturing Hardware` selected:
```
jquery.min.js:4 GET http://192.168.1.120:8090/i18n/en-GB.json 404 (Not found !)
send @ jquery.min.js:4
ajax @ jquery.min.js:4
n.<computed> @ jquery.min.js:4
getJSON @ jquery.min.js:4
jsonMessageLoader @ jquery.i18n.messagestore.js:113
load @ jquery.i18n.messagestore.js:57
load @ jquery.i18n.messagestore.js:77
load @ jquery.i18n.js:158
load @ jquery.i18n.js:156
initTrans @ settings.js:33
(anonymous) @ settings.js:59
j @ jquery.min.js:2
fireWith @ jquery.min.js:2
ready @ jquery.min.js:2
I @ jquery.min.js:2
VM39:1 Uncaught SyntaxError: Unexpected token u in JSON at position 0
    at JSON.parse (<anonymous>)
    at buildSchemaPart (eval at globalEval (jquery.min.js:2), <anonymous>:37:31)
    at eval (eval at globalEval (jquery.min.js:2), <anonymous>:146:3)
    at Array.forEach (<anonymous>)
    at HTMLDocument.eval (eval at globalEval (jquery.min.js:2), <anonymous>:145:40)
    at j (jquery.min.js:2)
    at Object.add [as done] (jquery.min.js:2)
    at n.fn.init.n.fn.ready (jquery.min.js:2)
    at eval (eval at globalEval (jquery.min.js:2), <anonymous>:1:13)
    at eval (<anonymous>)
buildSchemaPart @ VM38:37
eval @ VM38:146
eval @ VM38:145
j @ jquery.min.js:2
add @ jquery.min.js:2
n.fn.ready @ jquery.min.js:2
eval @ VM38:1
globalEval @ jquery.min.js:2
text script @ jquery.min.js:4
ub @ jquery.min.js:4
x @ jquery.min.js:4
(anonymous) @ jquery.min.js:4
send @ jquery.min.js:4
ajax @ jquery.min.js:4
n._evalUrl @ jquery.min.js:4
domManip @ jquery.min.js:3
append @ jquery.min.js:3
(anonymous) @ jquery.min.js:3
n.access @ jquery.min.js:2
html @ jquery.min.js:3
(anonymous) @ jquery.min.js:4
j @ jquery.min.js:2
fireWith @ jquery.min.js:2
x @ jquery.min.js:4
(anonymous) @ jquery.min.js:4
load (async)
send @ jquery.min.js:4
ajax @ jquery.min.js:4
n._evalUrl @ jquery.min.js:4
domManip @ jquery.min.js:3
append @ jquery.min.js:3
(anonymous) @ jquery.min.js:3
n.access @ jquery.min.js:2
html @ jquery.min.js:3
(anonymous) @ jquery.min.js:4
j @ jquery.min.js:2
fireWith @ jquery.min.js:2
x @ jquery.min.js:4
(anonymous) @ jquery.min.js:4
load (async)
send @ jquery.min.js:4
ajax @ jquery.min.js:4
n.fn.load @ jquery.min.js:4
loadContent @ ui_utils.js:130
(anonymous) @ content_index.js:277
dispatch @ jquery.min.js:3
r.handle @ jquery.min.js:3
```

**Comment by Paulchen-Panther** [comment]
It looks like you compiled Hyperion.NG without V4L2.
You are using OSMC because this OS is not officially supported.
Please install the published version.

**Comment by hissingshark** [comment]
@Paulchen-Panther The board is a Vero4K so yes, OSMC is the native system.

I've had to compile Hyperion and Hyperion.NG for it myself for years.  V4L2 used to be a problem for the build so I dropped it.  Never needed it as its an AMLogic device, using the FB-grabber and AML-grabber, rather than an external USB device.

I only see RPi images in your releases assets, so I didn't think there was a published version for this platform (a bit like the ODROID-C2).
